File: java/StringLength.java
import java.util.Scanner;

public class StringLength {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter String: ");
        String str = scanner.nextLine();
        scanner.close();

        char[] ch = str.toCharArray(); // character array
        int length = 0;
        for(char c : ch){ //for each loop
            length++;
        }
        System.out.println("Length of text " + length );
    }
}

File: kt/StringLength.kt
import java.util.Scanner;

fun main() {
    

    val scanner = Scanner(System.in);
    System.out.println("Enter Text: ");
    var str = scanner.nextLine();
    
    
    //without using function
    var ch = str.toCharArray(); // character array
    var l = 0;
    for(c in ch){ //for each loop
        l++;
    }
    println("Length of text " + l );

    //using length() function
}